app/dashboards/external_link_dashboard.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "administrate/base_dashboard"
+
+class ExternalLinkDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    versions: Field::HasMany.with_options(class_name: "PaperTrail::Version"),
```

Comment:
We don't seem to be doing this in any other dashbaord, so I think this is out of place.

Reply:
- Removed.
- Also changed the class the dashboard inherits from.

Reply:
I still see this line

Reply:
@skng5 :point_up:

Reply:
I see it too. I missed a step pushing it to GitHub. It should be there now.